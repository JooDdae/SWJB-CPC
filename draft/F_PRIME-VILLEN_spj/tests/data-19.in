31187379013460335332593197385733761115385817577409171096563456126171979677658178792749274151015153132175596353621180695279964928039511355769839173420119737583611340079371924161733881266344099539550192573889180115797643752385819743613348777349811763925797155774511867967790115527119379562329533389655120291019267794614001310983874273433817133439421760777418423102139113834617079800958019281
