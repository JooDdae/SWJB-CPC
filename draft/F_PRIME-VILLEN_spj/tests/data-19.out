193
2 195 2 998 2 623 2 3 2 2 1 17 2 8 4 2 148 2 2 4 18 0 0 3 2 1069 4 3 4 4 80 0 4 29 848 618 1 18 4 8 22 19 4 852 996 401 1 4 13 3 26 3 0 3 2 0 2 1 0 4 3 17 827 808 1014 3 1 22 1192 1 1011 3 0 0 2 3 3 136 146 0 4 2 575 8 4 2 4 3 23 18 0 2 553 159 8 1 13 18 4 2 2 152 386 600 1181 726 1147 539 279 0 3 22 4 117 4 3 1 2 1069 25 14 18 2 2 93 4 4 70 141 969 1 3 22 4 0 3 3 4 943 285 163 998 0 3 3 1 4 0 8 2 22 739 10 3 2 2 2 24 846 308 171 1 19 22 89 551 2 29 1050 587 14 2 1026 2 2 85 82 4 111 4 4 13 1053 172 2 156 2 23 89 909 1008 761 1149 